src/ucp/wireup/select.c

```diff
@@ -900,13 +908,15 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_add_memaccess_lanes(
     /* Select best transport which can reach registered memory */
     snprintf(title, sizeof(title), criteria->title, "registered");
     mem_criteria.title           = title;
-    mem_criteria.local_md_flags  = UCT_MD_FLAG_REG | criteria->local_md_flags;
+    mem_criteria.local_md_flags  = criteria->local_md_flags;
```

Comment:
seems like `criteria->local_md_flags` are not set by any of the calling functions and looks redundant here